lib/models/action.dart

```diff
@@ -144,14 +137,11 @@ class ListAction implements Explorable {
     required this.time,
     required BuiltList<Api.Cause> causes,
     required Api.ActionTypeEnum actionType,
-    required CausesUser? userInfo,
-  })  : cause = Cause.fromApiModel(causes[0], userInfo),
-        isCompleted = userInfo?.completedActionIds.contains(id) ?? false,
+  })  : cause = Cause.fromApiModel(causes[0]),
```

Comment:
Just a suggestion, doesn't Flutter have:
```suggestion
  })  : cause = Cause.fromApiModel(causes.first()),
```